ResourceStatusSystem/Command/PilotCommand.py

```diff
@@ -140,14 +156,15 @@ def doCache(self):
       # You should never see this error
       return S_ERROR('"%s" is not  Site nor Resource' % element)
 
-    result = self.rmClient.selectPilotCache(site, ce)
+    result = self.rmClient.selectPilotCache(site=site, cE=ce)
```

Comment:
...and "ce" here.

Reply:
The signature of the ResourceManagementClient method has always been using cE=. This was most likely done by the original author to cope with capitalising/de-capitalising then mapping to MySQL column names. A similar problem as with the VO above.